p2p/agent_test.go

```diff
@@ -149,9 +155,9 @@ func TestUnicast(t *testing.T) {
 	for i := 0; i < n; i++ {
 		neighbors, err := agents[i].Neighbors(ctx)
 		r.NoError(err)
-		r.True(len(neighbors) > 0)
+		r.True(len(neighbors) >= n/3)
```

Comment:
in theory, clients should connect with other `n-1` neighbors, after long enough of p2p activity
given the order/timing of them establish connection, `n/3` is a proper number that consistently pass the test